### PR TITLE
Display list of items on the Homepage

### LINKED ISSUE #15: [4pt]  - Display list of items on the Homepage - Student A
Create the main part of the homepage that keeps the layout from the wireframe: 

Screenshot 2021-06-27 at 21 50 23

When the page loads, the webapp retrieves data from  the selected API and shows the list of items on screen with the basic data (e.g. title + image).

Prepare also "Comments" and "Reservations" buttons. They should be doing nothing - just being displayed.

This task **does not** include displaying the number of likes for each item.

This task **does not** include the counter of the items.

### PR DESCRIPTION
**Pull request by Lookingfor00**
In this pull request, I have been able to solve the [issue 15](https://github.com/Trast00/Capston-javascript/issues/15#issue-1516503426):
- Add `ListMovies.js` module
- Add `getList` method into `ListMovies.js` to `GET` list of movies from [tv maze API](https://www.tvmaze.com/api#show-main-information)
- Add `display` method into `ListMovies.js` module to load dynamically the list of movies
- Add basic structure for page navigation

### CODE REVIEW

File: src/modules/ListMovies.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  display = () => {
+    this.list.forEach((movie) => {
+      // dynamic load of list of movies
+      const liMovies = document.createElement('li');
+      liMovies.classList.add('flex-center', 'movie');
+      liMovies.id = `movie-${movie.id}`;
+
+      const wrapper = document.createElement('div');
+      const img = document.createElement('img');
+      img.src = movie.image.medium; // `${this.urlApi}${movie.id}/images`
```
Comment (quavo19) [inline src/modules/ListMovies.js:18]: I know you have spent a lot of time and effort on this project and so forgot some basic mistakes 😄  , Please avoid using inline code as comments 
it makes your work look messy and unprofessional. liters don't usually catch these types of mistakes so try to go through your work the search for such mistakes

Comment (quavo19) [inline src/modules/ListMovies.js:18]: I know you have spent a lot of time and effort on this project and so forgot some basic mistakes 😄 , Please avoid using inline code as comments
it makes your work look messy and unprofessional. liters don't usually catch these types of mistakes so try to go through your work the search for such mistakes

File: src/style.css [hunk 2]
```diff
@@ -42,7 +42,69 @@
...
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+.list-movies {
+  width: 90%;
+  list-style: none;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
```
Comment (quavo19) [inline src/style.css:56]: great way to display your elements always Keep It Simple 👍 keep it up, using short lines of code makes your work neat and your program run faster 🥳

**Review by quavo19 — APPROVED** [review]

File: src/style.css (hunk 2 above)
Comment (Trast00) [reply src/style.css:56]: Thank you, I made the necessary change 👍

File: src/modules/ListMovies.js (hunk 1 above)
Comment (quavo19) [inline src/modules/ListMovies.js]: I know you have spent a lot of time and effort on this project and so forgot some basic mistakes 😄 , Please avoid using inline code as comments
it makes your work look messy and unprofessional. liters don't usually catch these types of mistakes so try to go through your work the search for such mistakes

**Review by quavo19 — COMMENTED** [review]
# 🟢 Project Approved! 🏆

## Hi  @Trast00  👋

Your project is complete! ✔️ 🏁

Your code was professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉
comp-2

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- ✔️ code is DRY 

Cheers and Happy coding!👏👏👏

## OPTIONAL 📝
 check the comment below


- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/ListMovies.js (hunk 1 above)
Comment (Lookingfor00) [reply src/modules/ListMovies.js]: Thank you for that suggestions, the change has been made 👍

**Pull request merged**